magda-web-client/src/UI/QualityIndicator.js

```diff
@@ -34,7 +38,14 @@ function QualityIndicator(props) {
                 {qualities.map((q, i) => (
                     <span key={i} style={getBarColor(i)} />
                 ))}
-            </span>
+            </span>&nbsp;
+            <div className="tooltip">
+                <img src={helpIcon} alt="Help Link"></img>
+                    <span className="tooltiptext">
+                        Calculated using the <Link to="page/dataset-quality">Open Data scale</Link>
```

Comment:
Looks good - just make it `/page/dataset-quality` instead of `page/dataset-quality` :)